haskell/test/Lorentz/Contracts/Test/FA2.hs

```diff
@@ -844,7 +786,7 @@ fa2Spec fa2Originate = do
 
           withSender commonOperator $ lCallEP fa2contract (Call @"Transfer") transfers
 
-          validate $ Left fa2SenderHookUndefined
+          validate $ Left fa2SenderHookFailed
```

Comment:
Why was undefined hook error changed to hook failed error?

Reply:
To match `FA2_RECEIVER_HOOK_FAILED` error as is defined in FA2 spec 🤷

Reply:
But here the hook is undefined here on the reciever, so shouldn't it be the undefined error as per FA2?

Reply:
Ah, you're right, renamed back